iteration/iteration_test.go

```diff
@@ -212,3 +212,74 @@ func (test *TestIterationRepository) TestUpdateIteration() {
 	assert.Equal(t, changedStart, *updatedIteration.StartAt)
 	assert.Equal(t, changedEnd, *updatedIteration.EndAt)
 }
+
+func (test *TestIterationRepository) TestLoadChildren() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+	newSpace := space.Space{
+		Name: "Space To Test Listing of Iteration Children",
```

Comment:
Be on the safe side, space names need to be unique, as a nicer pratice, I usually append a randomly generated uuid.